Modules/HTTP/Sources/HTTP/Message/Request.swift

```diff
@@ -64,7 +64,7 @@ extension Request {
 
         switch body {
         case let .buffer(body):
-            self.headers["Content-Length"] = body.count.description
+            self.headers["Content-Length"] = "\(body.count)"
```

Comment:
```swift
    /// A textual representation of this instance.
    ///
    /// Instead of accessing this property directly, convert an instance of any
    /// type to a string by using the `String(describing:)` initializer. For
    /// example:
    ///
    ///     struct Point: CustomStringConvertible {
    ///         let x: Int, y: Int
    ///
    ///         var description: String {
    ///             return "(\(x), \(y))"
    ///         }
    ///     }
    ///
    ///     let p = Point(x: 21, y: 30)
    ///     let s = String(describing: p)
    ///     print(s)
    ///     // Prints "(21, 30)"
    ///
    /// The conversion of `p` to a string in the assignment to `s` uses the
    /// `Point` type's `description` property.
    public var description: String { get }
```